core/templates/dev/head/pages/story-viewer-page/story-viewer-page.mainpage.html

```diff
@@ -0,0 +1,126 @@
+      <[storyDescription]>
+    </div>
+    <div ng-if="storyIsLoaded">
+      <story-viewer-chapters-list ng-if="storyPlaythroughObject.getStoryNodeCount() > 0"
+                                  playthrough-object="storyPlaythroughObject">
+      </story-viewer-chapters-list>
+    </div>
+  </div>
+  <style>
+    html, body {
+      background: #eeeeee no-repeat center center fixed;
+      background-size: cover;
+    }
+    .oppia-story-viewer-container {
+      position: relative;
+    }
+    .oppia-story-viewer-container .oppia-story-title {
+      color: #01645c;
+      font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+      font-size: 3em;
+      margin-top: 0px;
+      margin-bottom: 25px;
+      padding-top: 66.5px;
+      text-align: center;
+    }
+    .oppia-story-viewer-container .oppia-story-description {
+      font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+      font-size: 1.2em;
+      text-align: center;
+    }
+    .oppia-story-path-section {
+      background: #fff;
+      border-radius: 2px;
+      box-shadow: 0 2px 4px rgba(0, 0, 0, 0.14), 0 2px 4px rgba(0, 0, 0, 0.23);
+      float: left;
+      max-width: 675px;
+      padding: 20px 20px 60px 20px;
+      position: absolute;
+      text-align: left;
+    }
+    .oppia-story-table {
+      margin: 0 auto;
+      max-width: 90%;
+      padding: 50px 20px 420px 20px;
+      position: relative;
+      text-align: left;
+      top: 35px;
+      width: 100%;
+    }
+    .oppia-card-preview-panel {
+      background: #fff;
+      border-radius: 2px;
+      box-shadow: 0 2px 4px rgba(0, 0, 0, 0.14), 0 2px 4px rgba(0, 0, 0, 0.23);
+      left: 50%;
+      padding: 30px 85px 70px 85px;
+      pointer-events: none;
+      position: fixed;
+      text-align: left;
+      transform: translateX(29%);
+    }
+    .story-mascot {
+      left: 40px;
+      position: absolute;
+      top: 40px;
+      width: 120px;
+    }
+
+    /* To avoid double shadows caused by overlaying preview card
+    over the blank preview card */
+    .exploration-summary-tile md-card {
+      box-shadow: none;
+    }
+    .mobile-path-segment {
+      margin: 0 auto;
+      width: 250px;
+    }
+    .mobile-lesson-icon {
+      position: absolute;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      width: 100px;
+    }
+
+    @media only screen and (max-width: 998px) {
+      .oppia-activity-summary-tile-mobile-background-mask {
+        align-items: center;
+        background: rgba(0,0,0,0.25);
+        display: flex;
+        height: 100vh;
+        justify-content: space-around;
+        width: 100vw;
+        z-index: -1;
+      }
+    }
+  </style>
+{% endblock %}
+
+{% block footer_js %}
+  {{ super() }}
+  <script src="/third_party/static/ckeditor-4.9.2/ckeditor.js"></script>
+  <script src="/templates/dev/head/mathjaxConfig.js"></script>
+  <script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```

Comment:
Do we actually use these libraries in story viewer?

Reply:
The mathJax files are not needed, will remove them, but if I don't include ckeditor.js, it shows an error: `Uncaught ReferenceError: CKEDITOR is not defined` in jquery.min.js.

Reply:
No, it was needed because I imported an external file which I now removed, done.